app/src/main/java/com/github/sdp/ratemyepfl/database/UserRepository.kt

```diff
@@ -16,12 +19,16 @@ class UserRepository @Inject constructor(db: FirebaseFirestore) : UserRepository
         const val USERNAME_FIELD_NAME = "username"
         const val EMAIL_FIELD_NAME = "email"
         const val PICTURE_FIELD_NAME = "picture"
+        const val TIMETABLE_FIELD_NAME = "timetable"
```

Comment:
I think it would be good to maybe have it less tightly coupled in the future so that when downloading other users' info we don't download their timetables as well. What do you think?

Reply:
It would indeed be a good idea, but I don't really see a good way to do this. We would need to fetch every attribute independently from a user, except the timetable attribute. The timetable String is most likely going to be around 200-300 characters and it is loaded for only a single user, so it doesn't seem so bad to me.